docs/transfer-flow.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Transfer FLOW
+description: How to transfer FLOW using the Flow Go SDK
+---
+
+This is an example of how to construct a FLOW token transfer transaction
+with the Flow Go SDK.
+
+Before using this example, read the [Build a Transaction guide](../building-transactions)
+for a general overview of how Flow transactions are constructed.
+
+## Cadence Script
+
+The following Cadence script will transfer FLOW tokens from a sender
+to a recipient.
+
+_Note: this transaction is only compatible with Flow Mainnet._
```

Comment:
Maybe link to the core contracts docs pages which list the contract addresses for the other networks.